Automattic/wp-calypso
# Themes: use block themes as recommended for FSE

Sites with core Full Site Editing enabled will populate the Recommend Themes area with block-based themes.

#### Changes proposed in this Pull Request

* use block themes for the Recommend Themes area rather than Starter Page Template themes (the default)

#### Testing instructions

Browse themes on a site with FSE enabled. You should see (as of this writing) only one theme in the Recommend Themes area: `tt1-blocks`.

Switch to a site without FSE. You should see the regular list of Starter Page Template-supporting themes under Recommended.

Switch back to the FSE site again. Again, only FSE themes in Recommend.

`tt1-blocks` should not show up in the "all themes" section in either case. If it does, somehow, we have a REST API-level bug.

Fixes #50053
